src/js/components/Button/propTypes.js

```diff
@@ -68,7 +68,7 @@ if (process.env.NODE_ENV !== 'production') {
     primary: PropTypes.bool,
     reverse: PropTypes.bool,
     secondary: PropTypes.bool,
-    size: PropTypes.oneOf(['small', 'medium', 'large']),
+    size: PropTypes.oneOf(['small', 'medium', 'large', 'xlarge']),
```

Comment:
Also noticing, that `kind` is not in propTypes definition, nor is it documented. Is that still purposeful?

cc: @jcfilben

Reply:
We do have it in typescript, so it feels like adding it here would be reasonable but maybe for sake of progressing this PR we can handle that in a small standalone PR (since it's an existing thing)/

Reply:
Yes, I was not intending it for this PR, rather a point of awareness and stimulate discussion.

Reply:
> Also noticing, that kind is not in propTypes definition, nor is it documented. Is that still purposeful?

Sorry for slow response, I must of missed this comment when I was originally tagged. Anyway I'm looking into this... It also looks like the TS definition for kind is 'string' but looking at the code I think it could be a 'string' or 'object'. I'm not sure why we aren't documenting this, I can check with Shimi and maybe get more context.